bin/orm_exporttopostgres

```diff
@@ -0,0 +1,341 @@
+        )
+
+import logging
+
+
+def gen_data_files_list(pint_data_repo=None):
+    
+    if pint_data_repo is None:
+        pint_data_repo = os.path.realpath(
+                             os.path.join(
+                                 os.path.dirname(os.path.abspath(__file__)),
+                                 "..", "..", "pint-data"))
+
+    logger.debug("Pint Data repo: %s", repr(pint_data_repo))
+
+    data_files_pattern = os.path.join(pint_data_repo, 'data', '*.xml')
+    logger.debug("Data Files pattern: %s", repr(data_files_pattern))
+
+    data_files = glob.glob(data_files_pattern)
+    logger.debug("Data Files: %s", repr(data_files))
+
+    return data_files
+
+def get_commit_date(data_file):
+    data_dir = os.path.dirname(data_file)
+
+    output = subprocess.run(["bash", "-c",
+                             "cd %s; git log -n1 --pretty='format:%s' "
+                             "--date='format:%s' %s" %
+                             (os.path.dirname(data_file), '%cd', '%Y%m%d',
+                              os.path.basename(data_file))],
+                             stdout=subprocess.PIPE)
+
+    return output.stdout.decode('utf-8')
+
+def extract_provider_data_rows(parent_node, child_name):
+    rows = []
+
+    for child_node in parent_node.findall(child_name):
+        row = {}
+        for attr, value in child_node.items():
+            if not value:
+                continue
+            if attr == 'type':
+                if 'smt' in value:
+                    attr_value = ServerType.update
+                else:
+                    attr_value = ServerType.region
+            elif attr == 'state':
+                attr_value = getattr(ImageState, value)
+            elif attr in ['deletedon', 'deprecatedon', 'publishedon']:
+                attr_value = datetime.strptime(value, "%Y%m%d").date()
+            else:
+                attr_value = value
+            row[attr] = attr_value
+        rows.append(row)
+
+    return rows
+
+def extract_provider_region_map_rows(parent_node):
+    rows = []
+
+    for environment in parent_node.findall('environment'):
+        env_name = environment.get('name')
+        for region in environment.findall('region'):
+            region_name = region.get('name')
+            row = dict(environment=env_name,
+                       region=region_name, 
+                       canonicalname=region_name)
+            rows.append(row)
+            for alternate in region.findall('alternate'):
+                row = dict(environment=env_name,
+                           region=alternate.get('name'), 
+                           canonicalname=region_name)
+                rows.append(row)
+        #con.commit()
+    return rows
+
+def extract_data_from_file(data_file, data_store):
+
+    commit_date = get_commit_date(data_file)
+    reg_map = None
+
+    provider = os.path.basename(data_file).split('.')[0]
+    data_store[provider] = dict(version=f"{commit_date}.0",
```

Comment:
If we have multiple commits a day, which is possible, we should bump the last digit instead of always hardcoding it to '.0'.

Reply:
As per the discussion above I believe this is also addressed in an appropriate fashion, with the comments around this code being updated to reflect that this version is a "suggested" one, rather than the version that will actually be set.